packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.js

```diff
@@ -70,25 +75,25 @@ class OverridesResolverPlugin {
         }
     }
 
-    toOverrideRelative(path) {
-        const override = this.findOverride(path)
-        return path.substring(override.length + 1)
+    toOverrideRelative(_path) {
```

Comment:
what's the meaning for the `_`, i think you are trying to mean something like this is a special formatted path or something but i wasn't sure, could you add comment if there is anything special to this path? thanks!

Reply:
Same https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1240#discussion_r1214619020

Reply:
I was just avoiding conflict with `import path from 'path'` what I was doing `import p from 'path'` was creating a problem with it being different across files, I'll just do `_path` => `filepath`